yamllint.yaml

```diff
@@ -1,3 +1,7 @@
 extends: relaxed
 rules:
   line-length: disable
+
```

Comment:
this is a bad merge too, this doesn't work in the config file and had to be added as a command line flag (elsewhere)

Reply:
Sorry, rolled back from master to fixed it